frontstage/templates/surveys/surveys-share/survey-select.html

```diff
@@ -77,8 +63,42 @@
         {% endif %}
     {% endwith %}
 
-    <h1>Which surveys do you want to share?</h1>
+    <h1 class="ons-u-mb-m">Share access to surveys</h1>
+
+    {% call onsPanel({
+            "classes": "ons-u-mb-m",
+        })
+    %}
+    <p>You will still have access to surveys you share access to. If you no longer need access to the survey, <a href="{{ url_for('account_bp.transfer_survey_overview') }}">transfer your surveys</a></p>
+    {% endcall %}
+    {%
+        call onsDetails({
+            "classes": "ons-u-mb-xl",
+            "id": "details-example-with-warning",
+            "title": "How do I share access to a survey?"
+        })
+    %}
+        <p>Choose the surveys you want to share access to.</p>
```

Comment:
This isn't the text in the collapsible element on figma.

To share access to a survey:
Choose the surveys you want to share access to.
Enter the email address of the person who will be responding to the surveys.
We will email them instructions to access the surveys.
Once we confirm their access, they will be able to respond to the surveys and share access with their colleagues.